packages/cli/package.json

```diff
@@ -29,6 +29,7 @@
     "@react-native-community/cli-server-api": "^6.0.0-rc.0",
     "@react-native-community/cli-tools": "^6.0.0-rc.0",
     "@react-native-community/cli-types": "^6.0.0-rc.0",
+    "@react-native-community/metro": "^6.0.0-rc.0",
```

Comment:
I would name the package `cli-plugin-metro`, since the name `metro` itself in that namespace is quite broad and doesn't necessarily address the package's purpose.